<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POI Exporter</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select output POI folder</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="fileLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Output format </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="outputFormatComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Input vector layer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsMapLayerComboBox" name="vectorComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Select category column (optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="categoryComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Default category name (This will become the file name)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="defaultCategoryLineEdit">
     <property name="text">
      <string>POI</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Select column to be used as the POI name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="poiNameComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Default POI name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="defaultPOILineEdit">
     <property name="text">
      <string>POI</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Optional random color hash visualization column</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="visualComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Optional comment column</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="commentComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Optional description column</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="descriptionComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Optional URL column</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="webLinkComboBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Optional symbol column</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="symbolComboBox"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
